### DISCUSSION TITLE
Is there a good tauri open source project?

### DISCUSSION THREAD

**Discussion by hanxiaohuivc** [tauri-apps/tauri#2601] [Q&A] [answered]
Hello everyone. 
Are there any more complex open source projects based on tauri?  I want to learn the characteristics of tauri in depth through actual projects, such as patterns.

**Comment by FabianLars** [comment, marked as answer]
There is a dedicated channel (`#didathing-examples`) for stuff like this on the tauri discord.
There are of course also the repo examples, but yeah.
And you can look through: https://github.com/tauri-apps/tauri/network/dependents?package_id=UGFja2FnZS0yMTE5ODkxMDAy

Btw, imo you shouldn't worry tooo much about patterns as long as they aren't revisited for the current tauri version (they still can give you ideas of course). Basically the best practice in that regard is: disable the whole allowlist and enable only the endpoints you need, if you actually need them.
